src/script/miniscript.h

```diff
@@ -13,8 +13,8 @@
 #include <string>
 #include <vector>
 
-#include <stdlib.h>
 #include <assert.h>
+#include <cstdlib>
```

Comment:
You'll have to do this upstream as well?

Reply:
Opened a change upstream: https://github.com/sipa/miniscript/pull/124.